docs/reference/operators/layer/ebpf.md

```diff
@@ -0,0 +1,40 @@
+---
+title: ebpf
+sidebar_position: 10
+---
+
+The eBPF operator handles the [ebpf layer](../../spec/oci.md#the-ebpf-layer)
+that contains an ELF file with the eBPF programs. This operator loads the
+programs into the kernel and attaches them to the different hooks as specified
+by the gadget developer.
+
+## Instance Parameters
+
+### `iface`
+
+Network interface to attach networking programs to.
+
+Fully qualified name: `operators.oci.ebpf.iface`
+
+### `trace-pipe`
+
+Print debug information generated by eBPF with `bpf_printk()` to the terminal.
+
+Fully qualified name: `operators.oci.ebpf.trace-pipe`
+
+### `map-fetch-interval`
+
+Interval in which to iterate over "iter maps", i.e. hashmaps marked with
+`GADGET_MAPITER()`
+
+Fully qualified name: `operators.oci.ebpf.map-fetch-interval`
+
+Default: `1000ms`
+
+### `map-fetch-count`
+
+Number of map fetch cycles - use 0 for unlimited.
```

Comment:
Also this one is only available for `GADGET_MAPITER` data sources.

Reply:
Added it.